tests/test_m2m.py

```diff
@@ -44,30 +44,30 @@ def setUp(self):
                 </creator>
                 <meta />
             </metadata>
-        """
+        '''
         self.agent_tree = objectify.fromstring(agent_xml)
 
-        pub_xml = b"""<?xml version="1.0" encoding="UTF-8"?>
+        pub_xml = b'''<?xml version='1.0' encoding='UTF-8'?>
             <metadata>
                 <publisher>
                     <name />
                     <location />
                 </publisher>
                 <meta />
             </metadata>
-        """
+        '''
         self.pub_tree = objectify.fromstring(pub_xml)
 
     def test_metadata_record_setup(self):
-        # test that the creation of a record creates a MetadataRecord object.
+        """Tests that the creation of a record creates a MetadataRecord object."""
 
         record = m2m.MetadataRecord('mphillips')
         self.assertIsInstance(record, m2m.MetadataRecord)
 
         s = etree.fromstring(bytes(record))
-        self.assertEqual(len(s.findall('meta[@qualifier="metadataCreator"]')), 1)
+        self.assertEqual(len(s.findall("meta[@qualifier='metadataCreator']")), 1)
 
-        creator_string = s.findall('meta[@qualifier="metadataCreator"]')[0].text
+        creator_string = s.findall("meta[@qualifier='metadataCreator']")[0].text
```

Comment:
I  probably would have left all of these `findall` lines as they were with the single quotes on the outside and double on the inside as that seems more consistent to me, but isn't a big deal.